pkg/remediator/watch/filteredwatcher.go

```diff
@@ -268,7 +268,7 @@ func (w *filteredWatcher) Run(ctx context.Context) status.Error {
 		// 1. false, error -> We were unable to start the watch, so exit Run().
 		// 2. false, nil   -> We have been stopped via Stop(), so exit Run().
 		// 3. true,  nil   -> We have not been stopped and we started a new watch.
-		started, err := w.start(resourceVersion)
+		started, err := w.start(ctx, resourceVersion)
```

Comment:
I just love how before this change, the ctx was just being dropped on the floor. Sure, it was being given to the hander, but the handler can't terminate the watch and only gets run when there's an event. So before this, the watch would just keep running until it timed out, or more likely, the app terminated, leaving the connection open...